deploy/helm/sumologic/values.yaml

```diff
@@ -6,6 +6,9 @@ fullnameOverride: ""
 ## Use the same namespace as namespaceOverride in 'kube-prometheus-stack.namespaceOverride' if Prometheus setup is also enabled
 namespaceOverride: ""
 
+## Compression for logs, metrics and events:
```

Comment:
Could you enumerate what the valid values for this field are, both here and in the README?

Reply:
I've added these options - `Compression types can be gzip, snappy, zstd or deflate`

Reply:
It looks like it could also take `""` or `none` values, but didn't want to call these out, since we always want this enabled.

Reply:
this is one thing I'm concerned with exposing compression configuration, it allows the customer to disable it.